index.html

```diff
@@ -11,21 +11,28 @@
 </head>
 
 <body>
+   
+
     <div class="wrapper">
-        <nav class="nav-bar-desktop">
-            <a class="logo" href="#">Hamma Azarok</a>
-            <ul class="nav-list">
-                <li class="nav-item"><a href="#portfolio">Portofolio</a></li>
-                <li class="nav-item"><a href="#about">About</a></li>
-                <li class="nav-item"><a href="#contact">Contact</a></li>
+    <!-- Create the Navigation menu this way -->
+    <header class="nav-header">
+        <a href="#" class="nav-log">
+            <img src="img/logo.png" />
+        </a>
+        <nav id="navbar">
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="#" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="#portfolio" class="nav-link">Portofolio</a></li>
+                <li class="nav-item"><a href="#about" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
             </ul>
-        </nav>
-        <header class="nav-bar">
-            <a href="#">
-                <p>Hamma Azarok</p>
+            <a href="#" class="hamburger">
+                <span class="src-only" hidden="true">Open  main menu</span>
+                <i class="fas fa-bars"></i>
             </a>
-            <nav> <img src="img/Enabled.png" alt="Logo"></nav>
-        </header>
+        </nav>
+    </header>
```

Comment:
Kindly, stick to the design provided for the padding of the `nav` bar it doesn't have any padding around it please edit that.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="411" height="50" alt="image" src="https://user-images.githubusercontent.com/34578631/173548819-dae54d56-c3cb-4f8d-ac1d-6c74b76df422.png"> | <img width="374" alt="image" src="https://user-images.githubusercontent.com/34578631/173548748-7ecb5006-a981-4b77-97ee-a9a42590882b.png"> |